tasks/simplify.py

```diff
@@ -0,0 +1,55 @@
+from luigi import Task, Parameter
+import json
+import os
+from util import get_logger
+from simplification import SimplifyGeometriesMapshaper, SimplifyGeometriesPostGIS
+
+
+TABLE_ID = 'tableid'
+SIMPLIFICATION = 'simplification'
+GEOM_FIELD = 'geomfield'
+FACTOR = 'factor'
+MAX_MEMORY = 'maxmemory'
+SKIP_FAILURES = 'skipfailures'
+
+OBSERVATORY_SCHEMA = 'observatory'
+SIMPLIFICATION_MAPSHAPER = 'mapshaper'
+SIMPLIFICATION_POSTGIS = 'postgis'
+
+LOGGER = get_logger(__name__)
+
+
+class SimplifyOne(Task):
+    table = Parameter()
+
+    def run(self):
+        with open(os.path.join(os.path.dirname(__file__), 'simplifications.json')) as file:
+            tables = json.load(file)
+
+            if self.table in tables:
```

Comment:
I would remove this check, and let an exception be thrown if the table passed to this task is not defined in the JSON file. It is an error after all, and I think in this case it is preferable to fail than to silently eat the error.